src/shared/utils/bundleAnalysis.spec.ts

```diff
@@ -97,3 +101,69 @@ describe('formatTimeToString', () => {
     })
   })
 })
+
+describe('localizeBundleSize', () => {
+  describe('size is less then one kilobyte', () => {
+    describe('size is a positive number', () => {
+      it('returns size in bytes', () => {
+        const result = localizeBundleSize(900)
+        expect(result).toBe(900)
+      })
+    })
+
+    describe('size is a negative number', () => {
+      it('returns size in bytes', () => {
+        const result = localizeBundleSize(-900)
+        expect(result).toBe(-900)
+      })
+    })
+  })
+
+  describe('size is greater then one kilobyte and smaller then one megabyte', () => {
+    describe('size is a positive number', () => {
+      it('returns size in kilobytes', () => {
+        const result = localizeBundleSize(10000)
+        expect(result).toBe(10)
+      })
+    })
+
+    describe('size is a negative number', () => {
+      it('returns size in kilobytes', () => {
+        const result = localizeBundleSize(-10000)
+        expect(result).toBe(-10)
+      })
+    })
+  })
+
+  describe('size is greater then one megabyte and smaller then gigabyte', () => {
+    describe('size is a positive number', () => {
```

Comment:
I like this nested describes to better ... describe a test 👏

Reply:
I mean ... the top level is more the weight(?) of the number ... rather than the actual sign of the number 🤦‍♂️

Reply:
Haha I think you misunderstood what I said, I was complimenting the test and meant more like "Oh! I like how we have nested describes to better inform us what test is about!"